package.json

```diff
@@ -69,6 +69,7 @@
 	"dependencies": {
 		"@neovici/cosmoz-utils": "^5.1.0",
 		"haunted": "^5.0.0",
+		"lit": "^2.7.0",
```

Comment:
why?

Reply:
I ran npm install and both package and package-lock were updated, I thought that it should be included in the PR then. I'll revert this then.

Reply:
We usually use `lit-html` and not `lit`. They are different packages.